jdl/jdl-importer.js

```diff
@@ -102,9 +102,9 @@ function makeJDLImporter(content, configuration) {
          *          - exportedApplications: the exported applications, or an empty list
          *          - exportedEntities: the exported entities, or an empty list
          */
-        import: () => {
+        import: (logger = console) => {
```

Comment:
you've forgotten to add the `logger` param

Reply:
done